package.json

```diff
@@ -323,22 +336,24 @@
     "tether": "1.4.7",
     "tether-drop": "https://github.com/torkelo/drop",
     "tinycolor2": "1.4.1",
+    "uplot": "1.6.16",
     "uuid": "8.3.0",
     "visjs-network": "4.25.0",
     "whatwg-fetch": "3.1.0"
   },
   "resolutions": {
     "caniuse-db": "1.0.30000772",
-    "underscore": "1.12.1"
+    "underscore": "1.12.1",
+    "@types/slate": "0.47.2",
+    "@microsoft/api-extractor-model": "7.10.3",
```

Comment:
These dependency resolutions have been added to fix how they've been listed for our forks of api-extractor and api-documenter. When built/distributed, those packages should have had any workspace ranges transformed to the appropriate version, but for whatever reason in the distributed package.json the versions are still listed as "workspace:*", which causes a problem for yarn 2. Hopefully we can fix the actual issue with the forks at some point, but for now this solves the problem.